modules/tinymce/src/themes/silver/main/ts/api/Settings.ts

```diff
@@ -50,6 +51,25 @@ const isMergeStyleFormats = (editor: Editor): boolean => editor.getParam('style_
 const getLineHeightFormats = (editor: Editor): string[] =>
   editor.getParam('lineheight_formats', '1 1.1 1.2 1.3 1.4 1.5 2', 'string').split(' ');
 
+const getContentLangugaes = (editor: Editor): ContentLanguage[] => {
```

Comment:
This doesn't match the requirements, as it's supposed to use the same languages as the existing spellchecker language menu which is:
```
const selectContentLanguages = 'en,es,fr,de,pt,zh';
```

Reply:
Also there is a typo in the spelling.
```
getContentLangugaes
               ||
getContentLanguages
```